ARCHITECTURE.md

```diff
@@ -60,12 +52,43 @@ Rojo has many data structures and their purpose might not be immediately clear a
 
 To learn more, read about [`memofs` architecture](crates/memofs/ARCHITECTURE.md).
 
+### LiveServer
+
+LiveServer underlies the [`serve` command](#the-serve-command) and provides the web server which clients (such as the plugin) can use to interface with [`ServeSession`](#servesession).
+
+There's two parts to the API, the UI and the API clients use.
+
+The UI provides information about the current [`DOM`](#rojotree) has, including metadata. It also shows the project name, up-time, and version its Rojo is on.
```

Comment:
```suggestion
The UI provides information about the current project's [tree](#rojotree), including metadata. It also shows the project name, up-time, and version its Rojo is on.
```
This feels more accurate and avoids us having to explain the difference between the actual DOM and RojoTree later.